src/Dfc.ProviderPortal.ChangeFeedListener/Services/CourseAuditService.cs

```diff
@@ -126,7 +123,7 @@ public async Task<CourseAudit> Audit(ILogger log, Document auditee)
             }
         }
 
-        public async Task RepopulateSearchIndex(ILogger log)
+        public async Task RepopulateSearchIndex(ILogger log, bool deleteZombies)
```

Comment:
```suggestion
        public async Task RepopulateSearchIndex(ILogger log, bool deleteZombies = false)
```
a default value might make the calling code slightly easier to read

Reply:
It felt like being explicit about the behaviour you want at the call site was a safer option here.

Reply:
I think it's fine because you still have to be explicit to activate deletion